docs/devguide.md

```diff
@@ -31,11 +34,11 @@ layout:
     │   └── hack                # Non-build related scripts
     │   └── jenkins             # Jenkins configuration
     │   └── pkg                 # Contrib golang code
-    ├── charts                  # Helm charts for deployment
-    │   └── catalog             # Helm chart for deploying the catalog
-    │   └── ups-broker          # Helm chart for deploying the user-provided service broker
     ├── docs                    # Documentation
     ├── pkg                     # Contains all non-"main" Go packages
+    ├── plugin                  # Plugins for API server
+    ├── scripts                 # Push images to registry
```

Comment:
@kibbles-n-bytes can (should) we move the deploy.sh into /build/ instead since its kind of related to our build stuff?  Feels odd this one scripts it all on its own

Reply:
@duglin Thanks for suggestion. I agree with it and will remove the scripts dir.